test/sanity/validate-modules/ignore.txt

```diff
@@ -341,6 +341,9 @@ lib/ansible/modules/cloud/docker/docker_service.py E322
 lib/ansible/modules/cloud/docker/docker_service.py E324
 lib/ansible/modules/cloud/docker/docker_service.py E325
 lib/ansible/modules/cloud/docker/docker_service.py E326
+lib/ansible/modules/cloud/docker/docker_swarm.py E322
+lib/ansible/modules/cloud/docker/docker_swarm.py E324
+lib/ansible/modules/cloud/docker/docker_swarm.py E325
```

Comment:
The documentation and/or module should be fixed instead of adding additional ignores. If the issue is in a common docs fragment, that should be fixed in a separate PR first.

Reply:
@mattclay PR is done : https://github.com/ansible/ansible/pull/38378
When this PR will be merged, I will rebase this one.

Reply:
@mattclay Rebase done included PR https://github.com/ansible/ansible/pull/38378